l10n_ch_states/__manifest__.py

```diff
@@ -22,15 +22,19 @@
     'name': 'Switzerland Country States',
     'category': 'Localisation',
     'summary': '',
-    'version': '9.0.1.0.0',
-    'author': 'copado MEDIA UG, Odoo Community Association (OCA)',
+    'version': '10.0.1.0.0',
+    'author': '''
+        copado MEDIA UG,
+        brain-tec AG,
```

Comment:
I don't see a significant enough addition to the module to set brain-tec as author here. Please remove this line.
While you contribute, you are not an author of the module.